samples/wifi/provisioning/internal/sysbuild.cmake

```diff
@@ -0,0 +1,8 @@
+# nRF54H does not support wifi-enterprise
+if(NOT SB_CONFIG_SOC_SERIES_NRF54HX)
+if(NOT "wifi-enterprise" IN_LIST ${DEFAULT_IMAGE}_SNIPPET)
```

Comment:
can either combine both lines into 1, or keep as-is but indent lines 3-7 with 2 spaces